### ISSUE TITLE
Malfunction when  using two iPhones at the Same Time / parallel

### ISSUE THREAD

**Issue by moody425** [corona-warn-app/cwa-app-ios#1469]

**Comment by Ein-Tim** [comment]
Hey @moody425 
Could you please give more details about the two iPhones:
Model, iOS Version, Exposure Logs ("Einstellungen"
->"Begegnungsmitteilungen"
->"Aktiv"->"Begegnungsüberprüfungen" ->"Überprüfungen exportieren"), was Bluetooth always activated on both devices/did you always carry both phones with you everywhere.

**Comment by nilskasseckert** [comment]
Same problem for me and many of my friends. Even with new iPhones (we use 3 iPhones in one household)

**Comment by Ein-Tim** [comment]
@nilskasseckert 
But I think those 3 phones are also sometimes on different places, e.g.:
one at home, one at the grocery store, one at an Restaurant, or do you always go everywhere together and take all your phones with enabled Bluetooth with you?

**Comment by nilskasseckert** [comment]
@Ein-Tim 
Of course not. But if I have 4 low risk encounters and the other two smartphones show 0, it seems a bit wrong to me. 

But it only happens sometimes.

**Comment by Ein-Tim** [comment]
@nilskasseckert 
This **could** (but this is only one out of many possibilities) happen because you regularly go somewhere where the other members of your household do not go. ￼
If you now meet the same person on 4 different days on your regular "trips" (and this person later tests positive) CWA will show 4 Encounters. 

But as stated above: This is only one possibility explanation 🙂

**Comment by moody425** [comment]
Hey gentlemen ,
 
Two iPhones  : X and 5SE ,
 IOS 1.4.1 
24/7 online
Always Carried Close together 
BT always activated 
Exposure Logs always active and Saved every day 
Actual After 6 days - 10 contacts on iPhone X ....Zero contacts on iphone 5SE (Only one single contact on iphoneSE within six weeks)

Kind regards 

Moody425 

Dieter Schlosser

zz nur sicher erreichbar per E

**Comment by Ein-Tim** [comment]
Hey @moody425 are the contacts red or green?

**Comment by moody425** [comment]
All Green one or two every day 
But ....exposure Logs differ - 
Several times one is marked „0“!!!
And two are marked „1“ but up to three times consecutive repeated!!!
Totaly confusing and false !!!
It looks like, That there have been only two contacts for one day and the other 8 contacts never happend!!!!!
I will Analyse four consecutive weeks and report.

**Comment by Ein-Tim** [comment]
@moody425 

Okay thanks, you could also share your logs here.
Please keep in mind that there was an Issue starting at iOS 13.7 until iOS 14.1 which made the Log always show 0 matches, although there were some (for more details see #1106). 

<details>
<summary>How to export Exposure Notificaiton Logs</summary>

**German:**

`Einstellungen` ➡️ `Begegnungsmitteilungen` ➡️ `Status von Begegnungsaufzeichnungen` ➡️ `Begegnungsüberprüfungen` ➡️ `Überprüfungen exportieren`

**English:**

`Settings` ➡️ `Exposure Notifications` ➡️ `Exosure Logging Status` ➡️ `Exposure Checks` ➡️ `Export Exposure Checks`
</details>

**Comment by ndegendogo** [comment]
@moody425 please be aware that neither duration nor distance of a contact is a high-precision measurement. The bluetooth low energy (BLE) beacons are read and stored only once every 5 minutes. For this reason it can well happen that a short-time contact is seen and logged on one device but not on the other, even if the two devices were always at the same location / in short distance to each other.

**Comment by moody425** [comment]
Hey negendogo ,
Do you realy beleave what you 
Are Writing ?
More than three month, Not one 
Day with the Same number of contacts on Both iPhones- 
So fucking useless ,  you agree?


Dieter Schlosser

zz nur sicher erreichbar per E-Mail

Mobil - schlechtes Netz - +39.3703394918

**Comment by Ein-Tim** [comment]
@moody425 

Sorry to jump in here again but since this App is designed and programmed with the highes level of Data Protection, we can not say more that @ndegendogo said. Take a look at #1234, maybe this will help you. 
More than we've done to now we can't do.

**Comment by ndegendogo** [comment]
@moody425 As I said, the underlying technology BLE is not high-precision. So in the "green" contacts you just cannot expect the same contacts shown.

 > So fucking useless ,  you agree?

Well - this depends of course on your expectations.
The app can *in some cases* warn you that you might have been exposed to the virus. Not more.
The app will *never* give you a guarantee to show *all* your risky contacts. It will *never* protect you on its own or *replace* the recommended precautions (like keeping your distance / fresh air / face masks).

I am using the app, and I take the green contacts as a hint to postpone my planned visit to elder people. And a red warning as a hint to get tested before I go to work.

**Comment by dsarkar** [comment]
Dear @moody425 and community,

Thank you for your posts. As requested by the @Ein-Tim can provide us the ENF logs and confirm that you are using CWA Version 1.5.3 and iOS 14.2, please. Many Thanks.

Best wishes,
DS

Corona-Warn-App Open Source Team

**Comment by heinezen** [comment]
Closed due to missing feedback.

----
Corona-Warn-App Open Source Team
